### DISCUSSION TITLE
isnan undefined

### DISCUSSION THREAD

**Discussion by erdchris** [snl-dakota/dakota#78] [Q&A] [answered]
Hello. I'm trying to build Dakota 6.18.0 on RHEL 8.8 with Boost 18.3 (and 18.1). I'm using Intel compiler 2022.2.0 with gcc 11.4.0 and cmake 3.28.0. With either Boost version I get to 52% then this:

dakota-6.18.0-public-src-cli/src/unit/test_tolerance_intervals.cpp (lines 520, 521 and 590):
error: identifier "isnan" is undefined
BOOST_CHECK( isnan(computed_dstien_*[k] );

**Comment by mlaudu** [comment, marked as answer]
I got the same error. I just edited the file and changed isnan to
std::isnan. I hope this works for you as well.


*=======================================================================================*
*Musa L. Audu, Ph.D.*

*Research Professor
Principal Investigator*,
Department of Biomedical Engineering                    Advanced Platform
Technology Center,
Wickenden Building 309                                             Room
B-AC 275,
Case Western Reserve University                             Cleveland VA
Medical Center,
10900 Euclid Ave.                                                        10701
East Blvd.,
Cleveland, Ohio 44106.                                               Cleveland,
Ohio 44106.
U.S.A.
U.S.A.

*Phone:* 216-791-3800 Ext. 63821

**Reply by erdchris** [reply]
Thank you.
